vmdb/app/controllers/application_controller.rb

```diff
@@ -388,8 +388,8 @@ def event_logs
     return if record_no_longer_exists?(@record)
 
     @lastaction = "event_logs"
-    obj = @record.class.to_s == "Vm" ? "vm" : "host"
-    bc_text = @record.class.to_s == "Vm" ? "Event Logs" : "ESX Logs"
+    obj = @record.kind_of?(Vm) ? "vm" : "host"
+    bc_text = @record.kind_of?(Vm) ? "Event Logs" : "ESX Logs"
```

Comment:
@chessbyte The above lines almost seem like a fix to a potential issue from the VM/Template or VM/Cloud splits, where we would ONLY identify records of the exact Vm class as VMs rather than any that are a sub class of Vm.